src/DynamoCore/DynamoCore.csproj

```diff
@@ -83,8 +83,8 @@ limitations under the License.
       <HintPath>..\packages\Newtonsoft.Json.8.0.3\lib\net45\Newtonsoft.Json.dll</HintPath>
       <Private>True</Private>
     </Reference>
-    <Reference Include="ProtoGeometry, Version=2.9.0.3837, Culture=neutral, processorArchitecture=AMD64">
-      <HintPath>..\packages\DynamoVisualProgramming.LibG_225_0_0.2.9.0.3837\lib\net48\ProtoGeometry.dll</HintPath>
+    <Reference Include="ProtoGeometry, Version=2.11.0.4156, Culture=neutral, processorArchitecture=AMD64">
+      <HintPath>..\packages\DynamoVisualProgramming.LibG_225_0_0.2.11.0.4156\lib\net48\ProtoGeometry.dll</HintPath>
```

Comment:
Not that it matters much but we try to match this version with that for ASM227 (or whichever is the current latest).

Reply:
Sorry @aparajit-pratap I'm not sure I get it - LibG227 nuget had a different version number, and was pointing to a newer commit, I wanted all nugets to contain the same code... 

atleast I think thats what I did :) - grabbed everything on nuget from yesterday - 227 had an extra commit I believe.

Reply:
What I'm saying is that we should reference the protogeometry version that's coming from ASM227 here like we're doing in all other projects.

Reply:
ahh, okay, will make the change.